### ISSUE TITLE
Ability to move a discussion between forums / nodes

### ISSUE THREAD

**Issue by brendanheywood** [theCrag/website#2441]
eg if a discussion is started in the world node, but is really only relevant to a country or crag, we should be able to move it down. Likewise if a discussion was in a cliff and is more broadly relevant to the whole crag then move it up, or sideways or whatever.

I think a simple rule of thumb should be that if you have coordinator privileges in both the from and to node then you can move it. Very few people have coordinator role at the country and world level so that's fine.

I don't think we should worry about notifications too much in the transition, I think to make things simple only new messages to the discussion get notified to the new set of watchers, and we don't notify anyone when it moves.

See also #1365

**Comment by brendanheywood** [comment]
Closed #794 as dup of this

**Comment by brendanheywood** [comment]
+1 wanted this today.

Also has a soft dependency on #1476

**Comment by brendanheywood** [comment]
@scd I think this is actually the first forum task to run with, it's quite small and quick, and it will enable us to deal with issues more easily which may get worse when start implementing some of the other changes.

**Comment by brendanheywood** [comment]
+1 again from support email

**Comment by scd** [comment]
Also we need to resolve whether or not we allow private threads in a node. https://github.com/theCrag/website/issues/3162

**Comment by brendanheywood** [comment]
I think moving a thread is really important, probably the majority of threads are poorly located which spams people who don't want to see it. So is the decision around private thread being linked to a node blocking this?

**Comment by scd** [comment]
Yes I know and has put in in my list to get done today. I kind of time
boxed the work and got stuck with the public private logic. Also streams
causes another headache.

Note that a standard forum discussion only has one record to update, but a
node forum discussion has a second and the streams to update.

It is more than a hours work so I have put it back in the mix until I find
time again.

**Comment by brendanheywood** [comment]
Can we consider them each in turn?

A normal thread again a node is the key one to move, this seems simpler.

A discussion against a stream even against a node, I'm not it makes sense to ever move this? Sometimes you get a discussion which is off topic, but in this case it's probably best to start a new thread rather than attempt to move it

**Comment by brendanheywood** [comment]
+1

**Comment by brendanheywood** [comment]
+1 https://www.thecrag.com/discussion/2417509746

**Comment by brendanheywood** [comment]
+1 https://www.thecrag.com/discussion/2752900284/le-bouton-in-lphant-fontainebleau#m2754208881
